.env

```diff
@@ -0,0 +1,2 @@
+CODEPUSH_ANDROID_DEPLOYMENT_KEY=
+CODEPUSH_IOS_DEPLOYMENT_KEY=
```

Comment:
Do we need to put the staging keys here? Or is that something we don't want to push up to the repo?